imgix/urlhelper.py

```diff
@@ -2,9 +2,17 @@
 
 import hashlib
 import urllib
-import urlparse
 
-from constants import *
+try: # Python 3
```

Comment:
The six module might be helpful here. Not sure if it's worth introducing a dependency.

http://pythonhosted.org/six/